build.win32x86/third-party/Makefile.freetype2

```diff
@@ -20,6 +20,7 @@ $(FREETYPE2ARCHIVE):
 	
 $(THIRDPARTYLIBDIR)/$(FREETYPE2LIBNAME): $(FREETYPE2ARCHIVE)
 	tar x -f $(FREETYPE2ARCHIVE) -C $(THIRDPARTYDIR)
+	patch -p1 -d $(THIRDPARTYDIR) < ../../third-party/freetype291.patch
```

Comment:
I've put the patch file next to the spec file, so its easily noticed to delete the next time the spec is upgrade.